images/common/utils.sh

```
@@ -96,15 +96,15 @@ function wait_nginx_services {
 	set -e # Restore previous error setting.
 }
 
-function ssl_http_behaviour {
+ssl_http_behaviour() {
 	if [ "$NGINX_HTTP_ALLOW" == "True" ]; then
```

Shellcheck (0.11.0)

[warning] In POSIX sh, == in place of = is undefined.

(SC3014)